src/hardware/gameblaster.cpp

```diff
@@ -151,6 +153,8 @@ void GameBlaster::Open(const int port_choice, const std::string& card_choice,
 	assert(devices[1]);
 
 	is_open = true;
+
+	MIXER_UnlockMixerThread();
```

Comment:
Can't we reduce the scope of the lock a bit so the locking starts right above `MIXER_AddChannel`?